src/docs/implementations/influx.adoc

```diff
@@ -32,7 +31,7 @@ Or in Maven:
 </dependency>
 ----
 
-Metrics are rate-aggregated and pushed to Influx on a periodic interval. Rate aggregation performed by the registry yields datasets that are quite similar to those produced by Telegraf.
+Metrics are rate-aggregated and pushed to Influx on a periodic interval. Rate aggregation performed by the registry yields datasets that are quite similar to those produced by Telegraf. The following example configures Influx:
```

Comment:
Should we say InfluxDB instead?
Also: "pushed to Influx"

Reply:
Maybe. Probably. The line protocol might be pushed to telegraf as a buffer before it goes to InfluxDB. Also the name of the class is `InfluxMeterRegistry` so we have to pick which thing to be consistent with. Either way, I would qualify that the example configures a registry or meter registry for InfluxDB, rather than an InfluxDB instance itself.

Reply:
I looked it up, and it's InfluxDB. We should use the brand name, partly so that our readers will recognize and partly out of courtesy to the folks who make and partly for legal reasons. I changed  it to "The following example configures a meter registry for InfluxDB:" I also changed all the instances of "Influx" in the file to "InfluxDB" (so long as they weren't in a code block or otherwise part of some of other string - so when it was standalone in text) to get the branding correct.